course_contents/2_intro_to_python/notes/02. Loops.md

```diff
@@ -0,0 +1,159 @@
+### Loops
+- loops are used to run blocks of code multiple times
+- there are two kinds of loops in Python:
+    1. while loops
+    2. for loops
+
+**Extra resources**
+[What is a loop?](https://python.tecladocode.com/3_continuous_countries/1_loops.html#what-is-a-loop)
+
+### The `while` loop
+- while loop runs an indefinite number of times i.e. while loop runs until the given conditional becomes False
+- syntax for writing the while loop:
+    1. use the `while` keyword at the beginning of the loop
+    2. the keyword should be followed with a correctly written conditional with a colon at the end of it
+    3. indented block of code that will run until the loop breaks
+
+An example of the properly written `while` loop:
+
+```py
+counter = 10
+while counter > 0:
+  print(f"{counter} seconds left until New Year!")
+  counter -= 1
+print("Happy New Year!")
```

Comment:
if you want to show this example, how about using the `sleep()` function for a realistic use?

Reply:
Why would we mention the sleep function in the part of notes where we explain loops? I could change the example, but again, I went for simplicity 
If @jslvtr wants I can make more realistic examples, although I don't see a quick refresher benefiting much from it